### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by Aniketrajput039** [arunponnusamy/gender-detection-keras#7]
Which dataset you have used sir..?
And what accuracy you got..?
I also trained one but giving training set 96 % acc  but the  predictions are not satisfying to me.?

**Comment by Vanilla0921lu** [comment]
Hello,the model on my dataset are not satisfying to me,So I want to train my dataset,But thers is an error on it. Like this "cv2.error: OpenCV(4.1.1) /io/opencv/modules/imgproc/src/resize.cpp:3720: error: (-215:Assertion fail",So I want to know how big is your image, are they the same? how do you process your dataset? Thanks!
